pkg/config/config.go

```diff
@@ -82,6 +82,11 @@ type Config struct {
 	WebhookSecretName            string
 	WebhookSecretNamespace       string
 	WebhookSecretVolumeMountPath string
+
+	// LogFullBootstrapConfigAndCustomizationSpecs logs the entire bootstrap ConfigSpec
+	// and CustomizationSpec. Otherwise, fields that may contain sensitive information
+	// like passwords will not be logged. Defaults to false.
+	LogFullBootstrapConfigAndCustomizationSpecs bool
```

Comment:
Consider renaming to `LogSensitiveData` so it can include any other types of sensitive data in the future.